frontend/src/lib/components/Link.js

```diff
@@ -7,6 +7,10 @@ export function Link({ to, preventClick = false, ...props }) {
             href={to || '#'}
             {...props}
             onClick={(event) => {
+                if (event.ctrlKey) {
```

Comment:
Could you do `if (event.metaKey || event.ctrlKey)` so that it works on mac as well?

Reply:
sure will add it right away